app/views/analyse/replay.scala

```diff
@@ -10,11 +10,16 @@ import lila.app.UiEnv.{ *, given }
 import lila.common.Json.given
 import lila.round.RoundGame.secondsSinceCreation
 
+implicit val ec: scala.concurrent.ExecutionContext = scala.concurrent.ExecutionContext.global
```

Comment:
No, this is a view, and `ExecutionContext` is for I/O. We never, ever, do I/O in views.